python/ray/tune/suggest/nevergrad.py

```diff
@@ -111,6 +126,12 @@ def __init__(self,
         self._opt_factory = None
         self._nevergrad_opt = None
 
+        if points_to_evaluate is None:
+            self._points_to_evaluate = None
+        else:
+            assert isinstance(points_to_evaluate, (list, tuple))
+            self._points_to_evaluate = points_to_evaluate
+
```

Comment:
Let's raise an actionable error message here.I'd suggest something like:

```
if not isinstance(points_to_evaluate, Sequence):
    raise ValueError(
        f"Invalid object type passed for `points_to_evaluate`: {type(points_to_evaluate)}. "
        f"Please pass a list of points (dictionaries) instead.")
self._points_to_evaluate = list(points_to_evaluate)
```

(This would have to be linted)

Reply:
done